lib/index.js

```diff
@@ -241,12 +241,16 @@ Segment.prototype.normalize = function(msg) {
   msg.userId = msg.userId || user.id();
   msg.anonymousId = user.anonymousId();
   msg.sentAt = new Date();
+  // Add _metadata.
+  var failedInitializations = this.analytics.failedInitializations || [];
```

Comment:
Do we need to the gate this like with `this.options.addBundledMetadata` (i.e. are there customers who might not want this by default?i).

My thinking is no - but it would be great to double check. Two things in particular I'd want to confirm:
1. Will this create new columns in warehouses?  (I don't think it will )
2. Will this be sent to webhook and direct integrations? I think it will, and hence it might be picked up by those tools if they're not explicit about what fields they use.

Even if the answer to both 1 and 2 is yes, we might decide the tradeoff is worth it and to go ahead with this anyway, but it might be a good idea to give success and CSM's a heads up.

Would be great to get @jgershen's thoughts on this I think.

Reply:
Hey @f2prateek - I agree that it seems cleaner to send `_metadata` as an empty object if there's nothing to send but yeah @tsholmes would definitely know what our reasoning was for gating it. Will defer to him.

Reply:
It won't show up in warehouses, as the `_metadata` field is filtered out. It will show up in webhooks, but this is an existing bug that needs to be fixed for the bundled metadata at some point as well.

We originally gated it so we could test sending it through for a few projects and make sure it didn't show up in the warehouses. We can do that again now to be safe if we want, but I think we're ok since this has been working for a while.